### DISCUSSION TITLE
printing Rustdesk otp with display of in Raspberry pi

### DISCUSSION THREAD

**Discussion by xsssung** [rustdesk/rustdesk#6811] [Q&A]
Hi there, 
I'm in a project to remote control raspberry pi with Rustdesk.

My plan is this.
I'll send raspberry pi device to other person, and when they connect the device I want to get ID (probably ID is static so far) and OTP of rustdesk. I'm not sure it is possible, but I want to make that 20x8 lcd display which is connected to raspberry pi prints ID and OTP. (I'm preparing if the environment doesn't have monitor just in case)

I'm trying to make python code to print out both information, I want to import the part which makes OTP, and just print out those arrays. Is it possible that I import this part of code of Rustdesk as module in python?

<img width="623" alt="1" src="https://github.com/rustdesk/rustdesk/assets/155771654/0e1c7c02-1ea7-41f5-a97d-0d6eb4f0c1c3">

**Comment by xlionjuan** [comment]
https://rustdesk.com/docs/en/client/

> Getting ID
```bash
sudo rustdesk --get-id
```
It seems they didn't provide parameter about OTP, but you're using python, I think you can write a code to generate OTP and set it though command.
